packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -1,19 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * @author xzf0587 <zhaofengxu@microsoft.com>
```

Comment:
I'm not sure what's the best practice, is it fine to add our emails to public repo? @LongOddCode

Reply:
synced with @LongOddCode , it is ok to add the email as the emails can be found in git log as well